source/renderer/app/components/profile/analytics/AnalyticsConsentForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, useCallback } from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { Link } from 'react-polymorph/lib/components/Link';
+import { ButtonSpinnerSkin } from 'react-polymorph/lib/skins/simple/ButtonSpinnerSkin';
+import classnames from 'classnames';
+import styles from './AnalyticsConsentForm.scss';
+import { Intl } from '../../../types/i18nTypes';
+import { messages } from './AnalyticsConsentForm.messages';
+import { CollectedDataOverview } from './CollectedDataOverview';
+
+interface AnalyticsConsentFormProps {
+  intl: Intl;
+  loading: boolean;
```

Comment:
```suggestion
  isLoading: boolean;
```

Reply:
Seems like we have some inconsistency in the codebase:

![image](https://user-images.githubusercontent.com/6684554/176178134-e8cfa1d1-3589-4e48-9e8f-9b7e10eb5735.png)

But I believe skipping the `is` prefix is the idiomatic React approach (which is then following DOM attribute naming convention - eg. `disabled`, `readonly`)

Reply:
Would you agree @shawnbusuttil to keep it as is, or would you still say I should change this?

Reply:
It's a small thing. But I'm in favour of `is` as it is a slightly cleaner way to describe states.